src/explore-education-statistics-frontend/src/pages/index.tsx

```diff
@@ -102,9 +102,18 @@ function HomePage() {
       <div className="govuk-grid-row govuk-!-margin-bottom-3">
         <div className="govuk-grid-column-two-thirds">
           <p>
-            Use these services to find specific performance and other
-            information about schools and colleges in England:
+            Use these services to find related information and other statistical
+            services provided by DfE:
```

Comment:
Should it be "the Department for Education (DfE)"?

Reply:
Updated